ppr-ui/src/components/parties/composables/useSecuredParty.ts

```diff
@@ -95,6 +95,22 @@ export const useSecuredParty = (props, context) => {
     return idx !== -1
   }
 
+  const hasMatchingSecuredParty = (addedParty: PartyIF): boolean => {
+    // store state without newly added party
+    const parties = cloneDeep(getAddSecuredPartiesAndDebtors.value.securedParties)
+    if (localState.partyBusiness === 'I') {
```

Comment:
This doesn't make a lot of sense to me, from an outside perspective. What does `I` mean? 

Looking in the code i see `partyBusiness.value ===  I` or `partyBusiness.value === B` but what do these values represent? 

If you know, it might be a good idea to create an enum representing what these values truly mean so it's more readable & understandable in the code.

Reply:
Changed to enums, you're right about it not making much sense. New implementation is much easier to read/follow

Reply:
So much better, thank you 👍